app/views/posts/_post.html.erb

```diff
@@ -13,10 +13,36 @@
 <p class="post-time-ago my-2"><%= time_ago_in_words(post.created_at) %> ago.</p>
 <p class="post-content"><%= post.content %></p>
 <div class="d-flex justify-content-between post-like-line">
-	<button class="trans-btn"><i class="fa fa-thumbs-o-up" aria-hidden="true"></i>  Like</button>
+	<ul class="d-flex pl-0">
+		<li>
+			<% unless current_user.likes.exists?(post_id: post.id) %>
```

Comment:
**[REQUIRED]**
* View-related logic can be encapsulated in View helpers.
In this case we could have a method in `app/helpers/posts_heper` with this method.
```
module PostsHelper  
  def liked?(post_id)
    current_user.likes.exists?
  end  
end
```
and then just call the method in the views like
`<% unless liked?(post.id) %>`

([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/section/no_logic_in_helpers_requirement.png) from Pathwright)

Reply:
Thanks @wathigo
We fixed this and resent the pull request for review